RUN_uncor.m

```diff
@@ -0,0 +1,77 @@
+% Copyright 2008 - 2021, MIT Lincoln Laboratory
+% SPDX-License-Identifier: BSD-2-Clause
+
+%% Instruction
+% This script demonstrates how to use the UncorEncounterModel class to
+% draw samples from an uncorrelated encounter model and 
+% generate independent aircraft tracks by rejection sampling the model.
+% The default model parameter file is for a conventional aircraft. If a
+% different model is used, the start distribution needs to be updated.
+
+%% Startup script
+    file_startup = [getenv('AEM_DIR_BAYES') filesep 'startup_bayes.m'];
+run(file_startup);
+
+%% Inputs
+file_name='uncor_1200only_fwse_v1p2.txt';
+
+if contains (file_name, '.txt')
+    % MIT ASCII model parameter file
+    parameters_filename = [getenv('AEM_DIR_BAYES') filesep 'model' filesep file_name];
+elseif contains (file_name, '.mat')
+    % Canadian model parameter file
+    parameters_filename = [getenv('CANADIAN_MODELS') filesep file_name];
+end
+% Number of sample / tracks
+n_samples = 1;
+
+% Duration of each sample / track
+sample_time = 210;
+
+% Random seed
+init_seed = 1;
+
+%% Instantiate object
+mdl = UncorEncounterModel('parameters_filename', parameters_filename);
+
+%% Set start distribution 
+% Preallocate new start distribution local to workspace
+% The size of the variable corresponds to the number of
+% variables in the initial network
+start = cell(mdl.n_initial, 1);
+
+% Force the model to sample from specific variable bins
+% These values are for the uncorrelated conventional aircraft models. If
+% using an unconventional model (i.e. gliders) or the due regard model,
+% update or comment out accordingly. Also comment out if you don't want to
+% define a start distribution.
+%To use Canadian models, please comment those out
+start{1} = 1; % Geographic domain - CONUS (G = 1)
```

Comment:
Remove `start` for Canadian specific file

Reply:
Done!